### ISSUE TITLE
Uber API Shutting Down?

### ISSUE THREAD

**Issue by dxfan227** [kyle-kelly/MMM-uber#11]
Just got an email telling me the API wont work soon. Anyone else get this message? Any work around planned?

**Comment by kyle-kelly** [comment]
I just got the same one. They haven't said what the changes will be and they haven't updated the docs yet... We'll just have to see what happens and react accordingly :(

**Comment by dxfan227** [comment]
Is there any update on this? I just launched my mirror for the first time in a long time and both uber and Lyft modules just stuck at " loading"

**Comment by kyle-kelly** [comment]
Unfortunately, no. They've shut down the public API so this project is stale for the moment.

**Comment by dxfan227** [comment]
Same for Lyft?


David B. Ruiz | Director of Recruitment
Global Talent Solutions, Inc.
Email: druiz@gtsrecruiters.com <mailto:druiz@gtsrecruiters.com>
Direct: (408) 641-9701
Web: www.gtsrecruiters.com <http://www.gtsrecruiters.com/>

**Comment by mintaow** [comment]
Yes. Super disappointed. Lyft also downed their API so basically no official way to get the estimated pricing data.
